vmp/api/customer_client.go

```diff
@@ -264,7 +268,8 @@ func (apiClient *CustomerAPIClient) GetCustomerDeliveryRegion(accountNumber stri
 // UpdateCustomerDomainURL -
 func (apiClient *CustomerAPIClient) UpdateCustomerDomainURL(accountNumber string, domainType int, url string) error {
 	// TODO: support custom ids for accounts
-	relURL := fmt.Sprintf("pcc/customers/domains/%d/url?idtype=an&id=%s&partnerid=%d", domainType, accountNumber, apiClient.PartnerID)
+	baseURL := fmt.Sprintf("pcc/customers/domains/%d/url?idtype=an&id=%s", domainType, accountNumber)
+	relURL := FormatURLAddPartnerID(baseURL, apiClient.PartnerID)
```

Comment:
It looks like PartnerID must to be provided in the config file.
And we should have a PartnerID null check otherwise, the api call doesn't make a sense.